app/models/defendant.rb

```diff
@@ -37,7 +37,9 @@ def offence_ids
     offences.map(&:id)
   end
 
-  def linked?
-    offences.map(&:maat_reference).compact.present?
+  def maat_reference
+    return nil if offences == [] || offences.first.maat_reference&.first == 'Z'
```

Comment:
```suggestion
    return nil if offences.empty? || offences.first.maat_reference&.first&.eql?( 'Z')
```

Reply:
is it ever possible to have different `maat_references` on offences?

perhaps there should be something like below. and if it ever returns more that one then an error should be raise
```
offences.map(&:maat_reference).uniq.compact
```

Reply:
**^^ not a blocker**

Reply:
just to mention that `string.first` is a [rails extension](https://api.rubyonrails.org/classes/String.html#method-i-first).
consider alternative `start_with?` ruby string class method which may be clearer:
```
offences.first.maat_reference.start_with?('Z')
```

Reply:
have added a commit to handle these @mpw5